dwitter/static/libs/loopstopper.js

```diff
@@ -0,0 +1,43 @@
+window.stopper = {
+
+  loops: [],
+  loopTimers: [],
+  TIME_OUT_LENGTH: 500,
+
+  // States:
+  // undefined/0 - unmonitored
+  // 1           - Currently being monitored
+  // 2           - Exited
+  // 3           - Infinite
+
+  reset: function() {
+    this.loops = [];
+    this.loopTimers = [];
+  },
+  restartLoop: function(loopId) {
+    if (!this.loops[loopId] || this.loops[loopId] == 2) {
```

Comment:
So this should pass if it's 0 or 2. Is the main point to avoid reseting a 3 to a 1 or a resetting the timer on a loop in state 1?

Reply:
Having found the way instrumented code looks, I realize this is to make sure it's not in state 1.

Logic could be made more obvious here (maybe just a comment)